layouts/django/external.html

```diff
@@ -17,7 +17,6 @@
 {% block govuk_logo %}
   <div class="header-branding">
     <a href="https://www.gov.uk" title="Go to the GOV.UK homepage" class="header-logo">
-      <img src="{% static 'images/crown-logo-black-2x.png' %}" alt="">
```

Comment:
I guess my point is that this line isn't found on http://mojular.github.io/examples/headers.html